schema/ignition.json

```diff
@@ -356,6 +356,12 @@
             "enable": {
               "type": "boolean"
             },
+            "enabled": {
+              "type": ["boolean", "null"]
+            },
+            "disable": {

```

Comment:
I thought you were going to contain all the logic in the `enabled` field, where the three possible states are `true`, `false`, and `null`, so did you mean to still include the `disable` field?